zeppelin-web/src/app/notebook/paragraph/result/result.controller.js

```diff
@@ -12,8 +12,6 @@
  * limitations under the License.
  */
 
-import moment from 'moment';
```

Comment:
Basic question. Don't we have any related package that we can remove?

Reply:
Hi @jongyoul thanks for your attention to this change.

I checked package usage within the code deleted, and there did not seem to be other opportunities for further optimization.

But, maybe I am not understanding your exact question?

Reply:
Some other javascript files at this level including [paragraph-controller.js](https://github.com/apache/zeppelin/blob/master/zeppelin-web/src/app/notebook/notebook.controller.js) and [notebook-controller.js](https://github.com/apache/zeppelin/blob/master/zeppelin-web/src/app/notebook/notebook.controller.js) also import `moment`.

Thanks for the clarification @jongyoul

Reply:
Thanks